httpobs/scanner/analyzer/headers.py

```diff
@@ -567,36 +584,31 @@ def http_referrer_policy(reqs: dict, expectation='referrer-header-enabled') -> d
         'result': None,
     }
 
-    values = ['no-referrer',
-              'no-referrer-when-downgrade',
-              'origin',
-              'origin-when-cross-origin',
-              'same-origin',
-              'strict-origin',
-              'strict-origin-when-cross-origin',
-              'unsafe-url']
+    directives = {'no-referrer': 'referrer-policy-header-no-referrer',
+                  'no-referrer-when-downgrade': 'referrer-policy-header-no-referrer-when-downgrade',
+                  'origin': 'referrer-policy-header-origin',
+                  'origin-when-cross-origin': 'referrer-policy-header-origin-when-cross-origin',
+                  'same-origin': 'referrer-policy-header-same-origin',
+                  'strict-origin': 'referrer-policy-header-strict-origin',
+                  'strict-origin-when-cross-origin': 'referrer-policy-header-strict-origin-when-cross-origin'}
 
     response = reqs['responses']['auto']
 
     if 'Referrer-Policy' in response.headers:
-        output['data'] = response.headers['Referrer-Policy']
+        output['data'] = response.headers['Referrer-Policy'][0:256]  # Code defensively
```

Comment:
Also need to check the <meta> tag, which will be an exciting adventure for you.  ;)